howfairis/requesting/get_from_gitlab.py

```diff
@@ -0,0 +1,10 @@
+from .get_from_gitlab_no_auth import get_from_gitlab_no_auth
+from .get_from_gitlab_with_auth import get_from_gitlab_with_auth
+
+
+def get_from_gitlab(url, url_type, apikeys=None):
+
+    if apikeys is None:
```

Comment:
This code is much shorter than the equivalent code for github. Is it complete?

Reply:
fixed it same as github now